l10n/en-US/viewer.properties

```
@@ -251,21 +251,21 @@ printing_not_ready=Warning: The PDF is not fully loaded for printing.
 web_fonts_disabled=Web fonts are disabled: unable to use embedded PDF fonts.
 
 # Editor
-editor_free_text.title=Add FreeText Annotation
-editor_free_text_label=FreeText Annotation
-editor_ink.title=Add Ink Annotation
-editor_ink_label=Ink Annotation
+editor_text.title=Text
+editor_text_label=Text
+editor_draw.title=Draw
+editor_draw_label=Draw
```

Here, and elsewhere, I'm not crazy about the new string-ids. Basically it seems preferable to me if the string-ids themselves are still more inline with the actual code, since without prior knowledge it'd not be immediately obvious that e.g. "editor_draw" refers to the Ink-annotation.

It's perhaps a bit less nice, but how about something like this (a pattern we already use elsewhere)?
```suggestion
editor_free_text2.title=Text
editor_free_text2_label=Text
editor_ink2.title=Draw
editor_ink2_label=Draw
```